lib/galaxy/structured_app.py

```diff
@@ -127,7 +127,7 @@ class StructuredApp(MinimalManagerApp):
     file_sources: ConfiguredFileSources
     genome_builds: GenomeBuilds
     job_metrics: JobMetrics
-    model: GalaxyModelMapping
+    model: ModelMapping
```

Comment:
```suggestion
```

Reply:
Sure, but that leads to 

```
lib/galaxy/webapps/galaxy/api/__init__.py:117: error: Argument 1 to "GalaxySessionManager" has incompatible type "ModelMapping"; expected "SharedModelMapping"  [arg-type]
        return GalaxySessionManager(app.model)
```
Trying to fix makes one go in circles. Ultimately, the issue is that the mixins have varying types for `model`

```
lib/galaxy/app.py:591: error: Definition of "model" in base class "StructuredApp" is incompatible with definition in base class "ConfiguresGalaxyMixin"  [misc]
    class UniverseApplication(StructuredApp, GalaxyManagerApplication):
    ^
```